home-service/src/main/java/com/softserveinc/ita/homeproject/service/dto/CreateUserDto.java

```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.homeproject.service.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+public class CreateUserDto extends BaseUserServiceDto{
```

Comment:
Does it really worth creating a base dto for users? If it is please rename `BaseUserDto`

Reply:
We can make it without base dto. I just wanted to make more abstraction. If it's redundant, I can get rid of it